packages/upload-api/src/consumer/has.js

```diff
@@ -6,6 +6,7 @@ import { Consumer } from '@web3-storage/capabilities'
  * @param {API.ConsumerServiceContext} context
  */
 export const provide = (context) =>
+  // @ts-ignore
```

Comment:
could you add a comment here justifying the ignore?

Reply:
I removed it! good catch. My local checker was in a weird state across my editor and local terminal and ci, but I think it'll work now https://github.com/web3-storage/w3up/pull/1004/commits/e9882a4c4a7507dcf22ea31e090d84dee376b999